clients/hadoopfs/src/main/java/io/lakefs/committer/DummyOutputCommitter.java

```diff
@@ -216,11 +221,29 @@ private void cleanupJob() throws IOException {
         }
     }
 
+    private boolean needsSuccessFile() {
+        String markSuccessfulJobs = FSConfiguration.get(conf, outputPath.toUri().getScheme(), Constants.OC_SUCCESSFUL_JOB_OUTPUT_DIR_MARKER, "auto");
+        if (markSuccessfulJobs.equals("auto")) {
+            return conf.getBoolean(FileOutputCommitter.SUCCESSFUL_JOB_OUTPUT_DIR_MARKER, true);
+        }
+        return Boolean.parseBoolean(markSuccessfulJobs);
```

Comment:
Maybe add a validation that `markSuccessfulJobs`'s value isn't something weird (neither `true` or `false`)